code/modules/wiremod/shell/airlock.dm

```diff
@@ -30,6 +30,9 @@
 /obj/machinery/door/airlock/shell/canAIHack(mob/user)
 	return FALSE
 
+/obj/machinery/door/airlock/shell/allowed(mob/user)
+	return isAdminGhostAI(user)
```

Comment:
this breaks one of the circuit components on airlock which determines whether a player is allowed to access an airlock or not.

Reply:
wait when would a player "not" be allowed to access the airlock as (as far as I know) there are no ways to customise the access of a circuit airlock (although I could be mistaken)

Reply:
yes there is, it's a circuit component that gets included in the airlock